<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-1KGUKBL" failures="1" tests="2" name="Error Validation Test" time="16.225" errors="0" timestamp="2024-03-15T12:50:28 IST">
  <testcase classname="frameworks.tests.ErrorValidationsTest" name="loginErrorValidation" time="1.318">
    <skipped/>
  </testcase> <!-- loginErrorValidation -->
  <testcase classname="frameworks.tests.ErrorValidationsTest" name="loginErrorValidation" time="1.692">
    <failure type="java.lang.AssertionError" message="expected [Login Successfully] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Login Successfully] but found [Incorrect email or password.]
at frameworks.tests.ErrorValidationsTest.loginErrorValidation(ErrorValidationsTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- loginErrorValidation -->
  <testcase classname="frameworks.tests.ErrorValidationsTest" name="productErrorValidation" time="0.0">
    <ignored/>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- Error Validation Test -->
